docs/first-steps/post-deployment.md

```diff
@@ -138,6 +136,32 @@ Similarly to the management user's MFA enabling step, you are switching from usi
 
 As a last step you need to make sure that `MFA_ENABLED` is set to `true` in the `build.env` file.
 
+## Re-Configure profiles  MFA
+
+In order to everything works correctly, you need to set your aws config profiles with the correct mfa device configuration.
+
+Run again  `leverage credentials configure`  with the flag `--fetch-mfa-device` and select the option of `skip credentials configuration`.
```

Comment:
@rodriguez-matias Matu I'll rephrase this text with the following

> For everything to work as expected, you need to set the aws configuration profiles to the corresponding mfa-device configuration.
> 
> Then run `leverage credentials configure` again with the `--fetch-mfa-device` flag and select the `skip credentials configuration` option.

Reply:
Also, the flag `--skip-access-keys-setup` could be used instead to avoid the interactive step! https://leverage.binbash.com.ar/user-guide/leverage-cli/reference/credentials/#options

Reply:
@angelofenoglio thanks for your input, I agree that this clarification could be useful for the user.

Reply:
done!